lingetic-nextjs-frontend/app/languages/[language]/results/page.tsx

```
@@ -27,12 +29,22 @@ interface ResultsPageParams {
   useEffect(() => {
     playAgainRef.current?.focus();
-  }, []);
 
-  const searchParams = useSearchParams();
-  const { language } = useParams<ResultsPageParams>();
+    // Prefetch questions for the next round
+    queryClient.invalidateQueries({ queryKey: ["questions", language] });
+    queryClient.prefetchQuery({
+      queryKey: ["questions", language],
+      queryFn: () => fetchQuestions(language, getToken),
+      staleTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)